src/engine/bpmcontrol.h

```diff
@@ -152,14 +150,14 @@ class BpmControl : public EngineControl {
     double m_dSyncTargetBeatDistance;
     double m_dSyncInstantaneousBpm;
     double m_dLastSyncAdjustment;
-    bool m_resetSyncAdjustment;
+    QAtomicInt m_resetSyncAdjustment;
     FRIEND_TEST(EngineSyncTest, UserTweakBeatDistance);
-    double m_dUserOffset;
+    ControlValueAtomic<double> m_dUserOffset;
 
     TapFilter m_tapFilter;
 
-    TrackPointer m_pTrack;
-    BeatsPointer m_pBeats;
+    TrackPointer m_pTrack; // is witten from an engine worker thread
```

Comment:
Typo. Use a single line comment for both elements. What about the other concurrently accessed members? We should at least group them together.